src/watchful.ts

```diff
@@ -148,7 +149,7 @@ export class Watchful extends Construct implements IWatchful {
   }
 
   public watchDynamoTable(title: string, table: dynamodb.Table, options: WatchDynamoTableOptions = {}) {
-    return new WatchDynamoTable(this, table.node.uniqueId, {
+    return new WatchDynamoTable(this, table.node.id, {
```

Comment:
I don't think we can substitute `uniqueId` with `id` since `id` might not be unique across different stacks.

I think to avoid breaking downstream users of cdk-watchful and avoid creating new resources, we should use the `Names.uniqueId` API instead:

```suggestion
    return new WatchDynamoTable(this, Names.uniqueId(table), {
```